keyboards/takashicompany/jourkey/keyboard.json

```diff
@@ -0,0 +1,51 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "Jourkey",
+    "maintainer": "takashicompany",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "D1", "pin_b": "D0"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["D4", "C6", "D7", "E6", "B4", "F5", "F6", "F7", "B5", "B1", "B3", "B2", "B6"]
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0062",
+        "vid": "0x7463"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
* `LAYOUT` macro is missing key widths so it displays correctly
* Missing a layout macro that has an iso key